docs/start/authentication/graphql-storefront.mdx

```diff
@@ -66,17 +66,226 @@ content-type: application/json
 </Tab>
 </Tabs>
 
+#### Customer access tokens
+
+A customer access token is unique to an individual user's account because it represents an authenticated storefront session for GraphQL requests. You can obtain and use a customer access token only for server-to-server requests. Therefore, you must use the customer access token with a regular storefront token. A customer access token becomes invalid on all devices when you log out of a single device. 
+
+<Callout type="warning">
+Do not use this token for browser-side or client-side requests.
+</Callout>
+
+##### Obtaining a customer access token
+
+There are two options to obtain a customer access token.
+
+1. Login mutation
+2. Login with a JWT
+
+###### Login mutation
+
+Enter your user email and password to use the login mutation. The login mutation will return a customer access token in response to login actions as part of the GraphQL body instead of a cookie header. From there, you can store the customer access token in the presentation layer's session management system and send it with future GraphQL requests.
```

Comment:
This behavior varies based on whether the request is made from a browser or not. If the login() mutation request is from a browser, we will not return the CAT in the body, and will instead set a cookie.

If the request does not come from a browser and instead is used in a server-to-server context, the token will be returned in the body.